dev/src/backoff.ts

```diff
@@ -211,7 +205,7 @@ export class ExponentialBackoff {
    * @private
    * @returns {number} The jitter to apply based on the current delay.
    */
-  _jitterDelayMs() {
-    return (Math.random() - 0.5) * this._jitterFactor * this._currentBaseMs;
+  private jitterDelayMs() {
```

Comment:
It could be, but I'd prefer to lease as:
- I don't want to introduce logic changes in this PR (at least no large ones).
- This is the same code we use in the Web client, and I would like to match its internal structure.